cylc/flow/task_job_mgr.py

```diff
@@ -522,9 +522,6 @@ def submit_task_jobs(self, workflow, itasks, curve_auth,
                 cmd, [len(b) for b in itasks_batches])
 
             if remote_mode:
-                host = get_host_from_platform(
-                    platform, bad_hosts=self.task_remote_mgr.bad_hosts
-                )
```

Comment:
This host-selection appears to already have been attempted above, I'm not sure why it's being done again?

Reply:
I agree it is not needed